safe-global/web-core
# fix: add module/transaction guard identicons

## What it solves

Resolves #748

## How this PR fixes it

Identicons are now shown for modules (and transaction guards as they are on the same settings page).

## How to test it

Open `rin:0xf2565317F3Ae8Ae9EA98E9Fe1e7FADC77F823cbD` and observe the identicons next to the modules.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/193029749-7b5222a8-9d1c-4282-9dce-94ba86898230.png)
